exp/addons/internal/controllers/clusterresourcesetbinding_controller.go

```diff
@@ -77,8 +78,31 @@ func (r *ClusterResourceSetBindingReconciler) Reconcile(ctx context.Context, req
 		// Error reading the object - requeue the request.
 		return ctrl.Result{}, err
 	}
-
-	cluster, err := util.GetOwnerCluster(ctx, r.Client, binding.ObjectMeta)
+	var clusterName = binding.Spec.ClusterName
```

Comment:
nit: add a comment explaining why
```suggestion
        // Before 1.4 cluster name was stored as ownerRef; this code migrated this value to 
        // the spec.clusterBame field if necessary.
	var clusterName = binding.Spec.ClusterName
```